x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.test.ts

```diff
@@ -424,9 +424,8 @@ describe('The metric threshold alert type', () => {
 
 const createMockStaticConfiguration = (sources: any) => ({
   enabled: true,
-  query: {
-    partitionSize: 1,
-    partitionFactor: 1,
```

Comment:
I guess that this is not used anywhere?

Reply:
yes, cleaned up some kibana config options that we aren't using

Reply:
🙌